config.go

```diff
@@ -84,6 +86,8 @@ var (
 
 	defaultBitcoindDir  = btcutil.AppDataDir("bitcoin", false)
 	defaultLitecoindDir = btcutil.AppDataDir("litecoin", false)
+
+	defaultTorPrivateKeyPath = filepath.Join(defaultLndDir, defaultTorPrivateKeyFilename)
```

Comment:
Similar comment here w.r.t to `v2` prefix.

Reply:
Fixed.